docs/src/misc.rst

```diff
@@ -114,6 +114,12 @@ API
     Returns the libuv version number as a string. For non-release versions
     "-pre" is appended, so the version number could be "1.2.3-pre".
 
+.. c:function:: void uv_replace_allocator(uv_malloc_func malloc_func, uv_free_func free_func)
```

Comment:
Please add typedefs for uv_malloc_func and uv_free_func where datatypes are defined.